mender/templates/secret-artifacts-storage.yaml

```diff
@@ -13,7 +13,7 @@ metadata:
 data:
 {{- if eq .Values.global.storage "aws" }}
   AWS_URI: {{ .Values.global.s3.AWS_URI | default .Values.global.url | default (ternary (printf "https://%s" .Values.api_gateway.service.name ) (printf "http://%s" .Values.api_gateway.service.name) (.Values.api_gateway.env.SSL)) | b64enc }}
-  STORAGE_BUCKET: {{ .Values.global.s3.AWS_BUCKET | b64enc }}
+  AWS_BUCKET: {{ .Values.global.s3.AWS_BUCKET | b64enc }}
```

Comment:
DEPLOYMENTS_STORAGE_BUCKET should work at least with current master

Reply:
Yes, before the next Mender release I'll make a backward-compatible modification